packages/twenty-front/src/modules/object-record/record-board/utils/get-dragged-record-position.util.ts

```diff
@@ -0,0 +1,16 @@
+import { isDefined } from '~/utils/isDefined';
+
+export const getDraggedRecordPosition = (
+  recordBeforePosition?: number,
+  recordAfterPosition?: number,
+): number => {
+  if (isDefined(recordAfterPosition) && isDefined(recordBeforePosition)) {
+    return (recordBeforePosition + recordAfterPosition) / 2;
```

Comment:
Since you return in your if, I think we can have something with less indentation and more readable like this. Wdyt?
```typescript
export const getDraggedRecordPosition = (
  recordBeforePosition?: number,
  recordAfterPosition?: number,
): number => {
  if (isDefined(recordAfterPosition) && isDefined(recordBeforePosition)) {
    return (recordBeforePosition + recordAfterPosition) / 2;
  }

  if (isDefined(recordAfterPosition)) {
    return recordAfterPosition - 1;
  }

  if (isDefined(recordBeforePosition)) {
    return recordBeforePosition + 1;
  }

  return 1;
};
```